ZarCare Automation/Test/PageActions/FamilyMember_Page.cs

```diff
@@ -80,55 +80,177 @@ public static void AddfamilyMemberDetail(string MemFirstName, string MemLastName
             }
 
             FamilyMemberPage.Web_SubmitBtn.Click();
-            Wait.WaitTillPageLoad();
-
-           string actualAddMemberMessage = FamilyMemberPage.Web_ActualSuccessMessage.Text;
+            
+            string actualAddMemberMessage = FamilyMemberPage.Web_ActualSuccessMessage.Text;
             Assert.AreEqual(MemAddSuccMessage, actualAddMemberMessage);
```

Comment:
Please change the assert.equal need to add assert.that

Reply:
Changes to assert That